src/core/libraries/np_trophy/np_trophy.cpp

```diff
@@ -4,13 +4,20 @@
 #include <unordered_map>
 
 #include "common/logging/log.h"
+#include "common/path_util.h"
 #include "common/slot_vector.h"
 #include "core/libraries/error_codes.h"
 #include "core/libraries/libs.h"
+#include "externals/pugixml/src/pugixml.hpp"
```

Comment:
You should use `#include <pugixml.hpp>` for compatibility with the system library.

Reply:
i will change it, thank you